Library/Formula/evas.rb

```diff
@@ -1,7 +1,7 @@
 require "formula"
 
 class Evas < Formula
-  homepage "http://trac.enlightenment.org/e/wiki/Evas"
+  homepage "https://docs.enlightenment.org/auto/eet/evas_main.html"
   url "http://download.enlightenment.org/releases/evas-1.7.10.tar.gz"
```

Comment:
Would you update `url` to use `https` as well?:
https://download.enlightenment.org/releases/evas-1.7.10.tar.gz